README.md

```diff
@@ -25,7 +25,7 @@ We acknowledge and thank Bisq for their efforts, but we think the Monero communi
 
 ## Status of the project
 
-At the moment Haveno is only a Proof of Concept. It's already possible to initiate a crypto <-> XMR trade, but the platform still needs a lot of work before being available for public use.
+At the moment Haveno is only a Proof of Concept. It's already possible to initiate a crypto <-> XMR and fiat <-> XMR trades, but the platform still needs a lot of work before being available for public use.
```

Comment:
trades should be singular:  "It's already possible to initiate a crypto <-> XMR and fiat <-> XMR trade, but ..."

Reply:
or this which is better: "It's already possible to initiate crypto <-> XMR and fiat <-> XMR trades, but ..."